python/feathub/processors/flink/table_builder/aggregation_utils.py

```diff
@@ -64,6 +61,38 @@ def from_feature(feature: Feature) -> "AggregationFieldDescriptor":
         )
 
 
+class SlidingWindowAggregationFieldDescriptor(AggregationFieldDescriptor):
```

Comment:
The key semantics of this class is to represent `AggregationFieldDescriptor` with specific window size, and the same optimization can be applied to over window, right?

Can you rename the class to match its semantics?